app/views/users/index.html.erb

```diff
@@ -1,3 +1,17 @@
-<h1>Users List</h1>
 
 
+
+<h1>List of users</h1>
+
+<section>
+    <% @users.each do |user| %>        
+          <a href= <%= "users/#{user.id}"%>>
+          <%= image_tag(user.photo)%>
+            <h2> <%= user.name %></h2>
+              Number of posts:
+              <%= user.posts_counter || 0 %>
+            </div>
+            </div>
+          </a> 
+    <% end %>
+</section>
```

Comment:
- [ ] Kindly try as much as possible to follow the wireframe designs. This is a project requirement, you can do more with the designs if you want, but we should keep the wireframe.
- [ ] As you can see from the routes on the images of the wireframes, the user's index action is the root path. Your project doesn't have a root path, kindly add a root path (the page, the app will display when users first visit the page.)

![wire](https://user-images.githubusercontent.com/53124548/230359658-de576ecd-b573-4677-94b5-7004e3d334d3.png)

![styles](https://user-images.githubusercontent.com/53124548/230359720-0f355388-6d3f-4952-9682-2cd3fa015aa0.png)